src/utils/api.js

```diff
@@ -4,8 +4,18 @@ const articleApi = axios.create({
     baseURL:'https://nc-positive-news.herokuapp.com/api'
 })
 
-export const getArticles = () =>{
-    return articleApi.get('/articles').then(({data})=>{
+export const getArticles = (slug) =>{
+    if(slug){
+    return articleApi.get('/articles',{params: {topic :`${slug}`}}).then(({data})=>{
         return data.articles
+    })} else {return articleApi.get('/articles').then(({data})=>{
```

Comment:
You don't need to worry about using an `else` here. if you invoke getArticles with a null value it will not add it to the query :)

Reply:
Hi yes, this is what i thought but when i did that and i tried to press the home button, it was passing the query through as ?topic=undefined which would then crash the app.